mobile/src/navigations/root/root.tsx

```diff
@@ -22,41 +22,53 @@ const screenOptions: NativeStackNavigationOptions = {
 
 const Root: React.FC = () => {
   const dispatch = useAppDispatch();
-  const { isSurveyCompleted, authenticatedUser, authenticatedUserDataStatus } =
-    useAppSelector(({ auth }) => {
-      return {
-        isSurveyCompleted: auth.authenticatedUser?.isSurveyCompleted,
-        authenticatedUser: auth.authenticatedUser,
-        authenticatedUserDataStatus: auth.authenticatedUserDataStatus,
-      };
-    });
+  const {
+    isSurveyCompleted,
+    authenticatedUser,
+    authenticatedUserDataStatus,
+    surveyPreferencesDataStatus,
+  } = useAppSelector(({ auth }) => {
+    return {
+      isSurveyCompleted: auth.authenticatedUser?.isSurveyCompleted,
+      authenticatedUser: auth.authenticatedUser,
```

Comment:
You already took the whole `authenticatedUser`, I think getting `isSurveyCompleted` extra time is redundant